### PR TITLE
Add sample for add check box content control

### PR DESCRIPTION
**Pull request by VijayadharshiniMathiyalagan**
Description:
Added sample for add check box content control

### CODE REVIEW

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIO;
+
+//Register Syncfusion license
+Syncfusion.Licensing.SyncfusionLicenseProvider.RegisterLicense("Mgo+DSMBMAY9C3t2UlhhQlNHfV5DQmBWfFN0QXNYfVRwdF9GYEwgOX1dQl9nSXZTc0VlWndfcXNSQWc=");
```
Comment (Suriya-Balamurugan) [inline Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:5]: Remove license key.

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+using (WordDocument wordDocument = new WordDocument())
+{
+    // Add a section and paragraph to initialize an empty document structure
```
Comment (Suriya-Balamurugan) [inline Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:9]: //Add one section and one paragraph to the document.

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    wordDocument.EnsureMinimal();
+
+    // Append the label "Gender: Female" next to the first checkbox
+    wordDocument.LastParagraph.AppendText("Gender:\tFemale ");
+    // Append an inline content control to add a checkbox for the checked box
```
Comment (Suriya-Balamurugan) [inline Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:14]: //Append checkbox content control to the paragraph.

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+//Register Syncfusion license
+Syncfusion.Licensing.SyncfusionLicenseProvider.RegisterLicense("Mgo+DSMBMAY9C3t2UlhhQlNHfV5DQmBWfFN0QXNYfVRwdF9GYEwgOX1dQl9nSXZTc0VlWndfcXNSQWc=");
+
+using (WordDocument wordDocument = new WordDocument())
+{
+    // Add a section and paragraph to initialize an empty document structure
+    wordDocument.EnsureMinimal();
+
+    // Append the label "Gender: Female" next to the first checkbox
```
Comment (Suriya-Balamurugan) [inline Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:12]: //Gets the last paragraph.
WParagraph paragraph = document.LastParagraph;
//Add text to the paragraph.

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    wordDocument.LastParagraph.AppendText("Gender:\tFemale ");
+    // Append an inline content control to add a checkbox for the checked box
+    IInlineContentControl checkInline = wordDocument.LastParagraph.AppendInlineContentControl(ContentControlType.CheckBox);
```
Comment (Suriya-Balamurugan) [inline Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:15]: Change variable name  as "checkBox"

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    wordDocument.LastParagraph.AppendText("\tMale ");
+    // Append an inline content control to add a checkbox for the unchecked option
+    IInlineContentControl uncheckInline = wordDocument.LastParagraph.AppendInlineContentControl(ContentControlType.CheckBox);
+    // Set the checked and unchecked states.
+    uncheckInline.ContentControlProperties.CheckedState = tickState;
+    uncheckInline.ContentControlProperties.UncheckedState = uncheckedState;
+    // Set the initial state of the "Male" checkbox to unchecked
+    uncheckInline.ContentControlProperties.IsChecked = false;
+    // Save the document to a file in DOCX format
+    using (FileStream outputStream1 = new FileStream(Path.GetFullPath(@"Output/Result.docx"), FileMode.OpenOrCreate, FileAccess.ReadWrite))
```
Comment (Suriya-Balamurugan) [inline Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:45]: Modify the proper variable name.

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    // Create a CheckBoxState for the checked state, using a tick symbol in the Wingdings font
+    CheckBoxState tickState = new CheckBoxState
+    {
+        Font = "Wingdings",
+        Value = "\uF0FE" // Unicode for the tick symbol (✓) in Wingdings
+    };
+    // Set the checked state of the checkbox content control to display the tick symbol when selected
+    checkInline.ContentControlProperties.CheckedState = tickState;
+    // Create a CheckBoxState for the unchecked state, using an empty box symbol in the Wingdings font
+    CheckBoxState uncheckedState = new CheckBoxState
```
Comment (Suriya-Balamurugan) [inline Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:25]: change the variable name as unTickState

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs (hunk 1 above)
Comment (VijayadharshiniMathiyalagan) [reply Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:5]: Addressed

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs (hunk 2 above)
Comment (VijayadharshiniMathiyalagan) [reply Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:9]: Addressed

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs (hunk 3 above)
Comment (VijayadharshiniMathiyalagan) [reply Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:14]: Addressed

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs (hunk 4 above)
Comment (VijayadharshiniMathiyalagan) [reply Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:12]: Addressed

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs (hunk 5 above)
Comment (VijayadharshiniMathiyalagan) [reply Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:15]: Addressed

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs (hunk 6 above)
Comment (VijayadharshiniMathiyalagan) [reply Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:45]: Addressed

File: Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs (hunk 7 above)
Comment (VijayadharshiniMathiyalagan) [reply Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs:25]: Addressed

**Review by Suriya-Balamurugan — APPROVED** [review]
No concern.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**